modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -34,15 +36,15 @@ class {{classname}}(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-{{#vars}}
+{{#allVars}}
         '{{name}}': '{{{datatype}}}'{{#hasMore}},{{/hasMore}}
-{{/vars}}
+{{/allVars}}
     }
 
     attribute_map = {
-{{#vars}}
+{{#allVars}}
         '{{name}}': '{{baseName}}'{{#hasMore}},{{/hasMore}}
-{{/vars}}
+{{/allVars}}
```

Comment:
Due to the way the deserializer works in the api_client, the parent values must be present in `swagger_types` and `attribute_map`